packages/grid/_modules_/grid/hooks/features/virtualization/useGridVirtualRows.ts

```diff
@@ -271,7 +271,8 @@ export const useGridVirtualRows = (
     setRenderingState({ virtualPage: 0 });
 
     if (windowRef && windowRef.current) {
-      windowRef.current.scrollTo(0, 0);
+      windowRef.current.scrollTop = 0;
+      windowRef.current.scrollLeft = 0;
```

Comment:
also any particular reason for this change?

Reply:
Yes, I have documented it in the PR's description.